app/domain/work_days.py

```diff
@@ -133,6 +133,7 @@ def add_mission(self, mission):
                 for a in activities
                 if (
                     not a.is_dismissed
+                    or not self.max_reception_time
```

Comment:
:red_circle: 
Je pense qu'il faut plutôt mettre

`                if (
                    not a.is_dismissed
                    or (self.max_reception_time and a.dismissed_at > self.max_reception_time)
                )
`